tests/core/test_selector.py

```diff
@@ -512,6 +512,26 @@ def test_select_models_missing_env(mocker: MockerFixture, make_snapshot):
             ["model* & ^(tag:tag1 | tag:tag2)"],
             {'"model3"'},
         ),
+        (
+            [("model1", "tag1", None), ("model2", "tag2", None), ("model3", "tag3", None)],
+            ["(model*)+"],
```

Comment:
The tests that involve `+` are not useful unless you actually specify parent dependencies.